src/app/public/modules/image/fixtures/image.component.fixture.ts

```diff
@@ -0,0 +1,17 @@
+import { Component } from '@angular/core';
```

Comment:
Import statements need to follow code style guidelines.
See: https://docs.blackbaud.com/skyux-team-docs/style#imports